File: CS-Examples/17_HeaderAndFooter/RemoveHeader.cs
using System;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using System.IO;
using Spire.Doc;
using Spire.Doc.Documents;

namespace RemoveHeader
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Load the document
            string input = @"..\..\..\..\..\..\Data\HeaderAndFooter.docx";            
            Document doc = new Document();
            doc.LoadFromFile(input);

            //Get the first section of the document
            Section section = doc.Sections[0];

            //Traverse the word document and clear all headers in different type
            foreach (Paragraph para in section.Paragraphs)
            {
                foreach (DocumentObject obj in para.ChildObjects)
                {
                    //Clear footer in the first page
                    HeaderFooter header;
                    header = section.HeadersFooters[HeaderFooterType.HeaderFirstPage];
                    if (header != null)
                        header.ChildObjects.Clear();
                    //Clear footer in the odd page
                    header = section.HeadersFooters[HeaderFooterType.HeaderOdd];
                    if (header != null)
                        header.ChildObjects.Clear();
                    //Clear footer in the even page
                    header = section.HeadersFooters[HeaderFooterType.HeaderEven];
                    if (header != null)
                        header.ChildObjects.Clear();
                }
            }

            //Save and launch document
            string output = "RemoveHeader.docx";
            doc.SaveToFile(output, FileFormat.Docx);
            Viewer(output);
        }
        private void Viewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB-Examples/17_HeaderAndFooter/RemoveHeader.vb
Imports System.Text
Imports System.IO
Imports Spire.Doc
Imports Spire.Doc.Documents

Namespace RemoveHeader
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			Dim input As String = "..\..\..\..\..\..\Data\HeaderAndFooter.docx"

			'Create a Word document
			Dim doc As New Document()

			'Load the file from disk
			doc.LoadFromFile(input)

			'Get the first section of the document
			Dim section As Section = doc.Sections(0)

			'Traverse the word document and clear all headers in different type
			For Each para As Paragraph In section.Paragraphs
				For Each obj As DocumentObject In para.ChildObjects
					'Clear footer in the first page
					Dim header As HeaderFooter
					header = section.HeadersFooters(HeaderFooterType.HeaderFirstPage)
					If header IsNot Nothing Then
						header.ChildObjects.Clear()
					End If
					'Clear footer in the odd page
					header = section.HeadersFooters(HeaderFooterType.HeaderOdd)
					If header IsNot Nothing Then
						header.ChildObjects.Clear()
					End If
					'Clear footer in the even page
					header = section.HeadersFooters(HeaderFooterType.HeaderEven)
					If header IsNot Nothing Then
						header.ChildObjects.Clear()
					End If
				Next obj
			Next para

			'Save the document
			Dim output As String = "RemoveHeader.docx"
			doc.SaveToFile(output, FileFormat.Docx)

			'Dispose the document
			doc.Dispose()
			
			Viewer(output)
		End Sub
		Private Sub Viewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace
